Dockerfile

```diff
@@ -80,6 +80,9 @@ RUN	apt update && apt install -yqq \
 	ffmpeg \
     	nodejs \
 	gstreamer1.0-tools gstreamer1.0-plugins-good gstreamer1.0-plugins-base gstreamer1.0-plugins-bad \
+	procps \
+	vnstat \
+	linux-tools-common linux-tools-generic linux-tools-5.4.0-148-generic \
```

Comment:
`linux-tools-generic` alone should be enough. it should pick up correct `linux-tools-<version>-generic` based on installed kernel version.

Reply:
Got it, updating.